poo/herencia/telefonoCamara/index.ts

```diff
@@ -0,0 +1,48 @@
+class Telefono{
+    private onOf: boolean;
+    protected bateriaActual:number;
+
+    public constructor(_onOf?:boolean){
+        (_onOf==null||_onOf==undefined)?this.onOf=false:this.onOf=_onOf;
+        this.bateriaActual= 0;
+    }
+    public prenderApagar():void{
+        (this.onOf==false)?this.onOf=true:this.onOf=false;
```

Comment:
Para hacer mas simple esta condicion podes hacer `this.onOf = !this.onOf` con el ! negas el valor actual